kornia/constants.py

```diff
@@ -1,83 +1,88 @@
 from enum import Enum, EnumMeta
-from typing import TypeVar, Union, cast
+from typing import Any, Iterator, Union
 
 import torch
-from torch import Tensor
+
+from kornia.core import Tensor
 
 __all__ = ['pi', 'DType', 'Resample', 'BorderType', 'SamplePadding']
 
 pi = torch.tensor(3.14159265358979323846)
-T = TypeVar('T', bound='ConstantBase')
 
 
-class ConstantBase:
-    @classmethod
-    def get(cls, value: Union[str, int, T]) -> T:  # type: ignore
-        if type(value) is str:
-            return cls[value.upper()]  # type: ignore
-        if type(value) is int:
-            return cls(value)  # type: ignore
-        if type(value) is cls:
-            return value  # type: ignore
-        raise TypeError()
+class EnumMetaFlags(EnumMeta):
+    def __iter__(self) -> Iterator[Enum]:  # type: ignore[override]
```

Comment:
can we verify that the type ignore in this region are needed ?

Reply:
I particularly didn't like this solution because I couldn't remove all the `#type: ignore`. However, yesterday I couldn't think of how to keep this logic in `constants.py`. This `# type: ignore[override]` is because of what mypy points to [Liskov substitution principle](https://mypy.readthedocs.io/en/stable/common_issues.html#incompatible-overrides). 

For this particular case, it is because it is not `compatible` with the `EnumMeta` type (because `Enum` only exists after `EnumMeta`). However, in order for mypy to understand the type when we interact by the "contants", it is necessary to have this overwriting here...